src/gsy_e/gsy_e_core/simulation/simulation.py

```diff
@@ -411,6 +411,9 @@ def _init(self) -> None:
 
         self.area.activate_energy_parameters(self.config.start_date)
 
+        if self.config.external_connection_enabled:
+            global_objects.scm_external_global_stats(self.area)
```

Comment:
Same comment as below, I think that it is better to embed the if block inside the body of the `scm_external_global_stats.__call__` in order to not have the if block in the simulation class